xwiki-platform-core/xwiki-platform-oldcore/src/main/java/org/xwiki/security/authservice/internal/StandardXWikiAuthServiceComponent.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.security.authservice.internal;
+
+import java.security.Principal;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.component.phase.Initializable;
+import org.xwiki.component.phase.InitializationException;
+import org.xwiki.security.authservice.XWikiAuthServiceComponent;
+
+import com.xpn.xwiki.XWikiContext;
+import com.xpn.xwiki.XWikiException;
+import com.xpn.xwiki.user.api.XWikiUser;
+import com.xpn.xwiki.user.impl.xwiki.XWikiAuthServiceImpl;
+
+/**
+ * The standard XWiki authenticator, to use by default.
+ * 
+ * @version $Id$
+ * @since 15.3RC1
+ */
+@Component
+@Singleton
+@Named(StandardXWikiAuthServiceComponent.ID)
+public class StandardXWikiAuthServiceComponent implements XWikiAuthServiceComponent, Initializable
+{
+    /**
+     * The identifier of the authentication service.
+     */
+    public static final String ID = "standard";
+
+    private XWikiAuthServiceImpl service;
+
+    @Override
+    public String getId()
+    {
+        return ID;
+    }
+
+    @Override
+    public void initialize() throws InitializationException
+    {
+        this.service = new XWikiAuthServiceImpl();
```

Comment:
Is the initialize required here (instead of instantiating `XWikiAuthServiceImpl` line 53)?

Reply:
No, not really. I think I used to have more stuff in this initialize and forgot to cleanup.